src/js_native_api_v8.cc

```diff
@@ -468,12 +468,14 @@ class Reference : public RefBase {
       // the reference itself has already been deleted so nothing to do
       return;
     }
+    reference->_secondPassParameter = nullptr;
     reference->Finalize();
```

Comment:
IIUC, the problem happens here as the `SecondPassCallParameterRef` ownership is been transferred to the SecondPass weak parameter but its content (`v8impl::Reference`) is already been destroyed by env teardown. I'm afraid I did not understand how the patch could fix the problem?

Reply:
@legendecas see comment above which explains how the fix addresses the issue.